DOCPLAN-30: Updates for assemblies to align with DITA migration (#102628)
openshift/openshift-docs: virt/post_installation_configuration/virt-post-install-storage-config.adoc

[IMPORTANT]
====

> HPP storage pools must not be in the same partition as the operating system. Otherwise, the storage pools might fill the operating system partition. If the operating system partition is full, this might negatively impact performance, or the node can become unstable or unusable.

Reviewer: ```suggestion
HPP storage pools must not be in the same partition as the operating system. Otherwise, the storage pools might fill the operating system partition. If the operating system partition is full, performance might degrade or the node might become unstable or unusable.
```

optional suggestion for clarity

Author: I'll leave this one as is since this tweak was just to appease Vale's passive vs active language requirements :) I feel like this would introduce the issue
